### PR TITLE
Travisci sonarcloud

### PR DESCRIPTION
**Pull request by morgantaschuk**

### CODE REVIEW

File: src/main/java/ca/on/oicr/gsi/visionmate/ServerConfig.java [hunk 1]
```diff
@@ -328,5 +328,5 @@ public void setSortOrder(SortOrder sortOrder) {
     if (sortOrder == null) throw new NullPointerException("Sort order cannot be null");
     this.sortOrder = sortOrder;
   }
-
+//FIXME
```
Comment (sqwci) [inline src/main/java/ca/on/oicr/gsi/visionmate/ServerConfig.java]: MAJOR Take the required action to fix the issue indicated by this comment. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1134)

**Comment by sqwci** [comment]
SonarQube analysis reported 5 issues
* ![CRITICAL][CRITICAL] 2 critical
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 2 minor

Watch the comments in this conversation to review them.

#### 4 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [ServerConfig.java#L152](https://github.com/oicr-gsi/visionmate-client/blob/e1805ab566494c4612d9567e6a94a39037c4d02f/src/main/java/ca/on/oicr/gsi/visionmate/ServerConfig.java#L152): Define a constant instead of duplicating this literal "Command must not be null" 6 times. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)
1. ![CRITICAL][CRITICAL] [ServerConfig.java#L153](https://github.com/oicr-gsi/visionmate-client/blob/e1805ab566494c4612d9567e6a94a39037c4d02f/src/main/java/ca/on/oicr/gsi/visionmate/ServerConfig.java#L153): Define a constant instead of duplicating this literal "Command must not be empty" 6 times. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [ServerConfig.java#L18](https://github.com/oicr-gsi/visionmate-client/blob/e1805ab566494c4612d9567e6a94a39037c4d02f/src/main/java/ca/on/oicr/gsi/visionmate/ServerConfig.java#L18): Remove this empty statement. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)
1. ![MINOR][MINOR] [ServerConfig.java#L18](https://github.com/oicr-gsi/visionmate-client/blob/e1805ab566494c4612d9567e6a94a39037c4d02f/src/main/java/ca/on/oicr/gsi/visionmate/ServerConfig.java#L18): Remove this redundant "static" qualifier. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2786)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/main/java/ca/on/oicr/gsi/visionmate/ServerConfig.java [hunk 2]
```diff
@@ -15,7 +15,10 @@
  */
 public class ServerConfig {
   
-  public static enum SortOrder {ROWS, COLUMNS};
+  public enum SortOrder {ROWS, COLUMNS};
```
Comment (sqwci) [inline src/main/java/ca/on/oicr/gsi/visionmate/ServerConfig.java:18]: MINOR Remove this empty statement. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)

**Comment by sqwci** [comment]
SonarQube analysis reported 1 issue
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by morgantaschuk** [comment]
ok to test

**Review by djcooke — APPROVED** [review]

**Pull request merged**